Week3Assinment1_Rajshukla #7.cpp

```diff
@@ -0,0 +1,60 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+   int arr[10],s,p,key,i,b=0,c=0,pos;
+   int found=0;
+   cout<<"enter size of array:";
+   cin>>s;
+   cout<<"enter array elements:";
+   for(i=0;i<s;i++)
+    {
+       cin>>arr[i];
+    }
+
+     cout<<"\n enter key:";
+     cin>>key;
+      for(i=0;i<s;i++)
+      {

```

Comment:
What is the worst case time complexity here ? If he array is sorted you will take o(n) the problem asks for o(log n)